upgrades/upgrade-1.6-2.0.rst

```diff
@@ -0,0 +1,296 @@
+Upgrading from Sulu 1.6 to 2.0
+==============================
+
+We provide a very extensive `UPGRADE.md file`_ in the ``sulu/sulu`` repository containing almost every breaking change
+we made during the development of Sulu 2.0. However, this is a huge list and most projects won't be affected by the
+majority of these breaking changes. Therefore thi sguide provides a list of the most important changes. By following
+the steps of this guide you should be able to update most Sulu 1.6 projects to Sulu 2.0. Please be aware that this
+guide does not contain any steps needed if your project overrides Sulu's internal mechanisms.
+
+**1. Create a new** `sulu/skeleton`_ **project and import your projects** ``.git`` **folder**
+
+In our experience it makes more sense and will save you future headache if you start over with a new project based on
+`sulu/skeleton`_ which is the preferred project template for starting a Sulu 2.0 project. The old `sulu/sulu-minimal`_
+and the deprecated `sulu/sulu-standard`_ project templates are not recommended for running a Sulu 2.0 project. You
+could also take your existing `sulu/sulu-minimal`_ project and restructure it to match the `sulu/skeleton`_ repository,
+but that is probably going to be more work than doing it the other way around.
+
+The recommended way to achieve this, is to create a new project using ``composer create-project`` and copy the old
+``.git`` folder to the root of the new project.
+
+.. code-block:: bash
+
+    composer create-project sulu/skeleton <project-name>
+    cp <old-project-path>/.git <project-name>
+
+Afterwards a ``git status`` in the new folder should reveal all the changes from your old repository to the new
+`sulu/skeleton`_ installation, since ``git`` is doing a very good job on recognizing the file changes here.
+
+.. note::
+    Theoretically these changes could already be commited now, but we recommend to do this at the end of this guide.
+    That will create a single commit containing the whole upgrade.
+
+**2. Copy over your old project files to the new project**
+
+This is probably the trickiest part of the upgrade. You have to move all of your project specific files from the old
+project folder to the new one.
+
+These files are not always located in the same place as previously. Use the below table to get an idea how to move
```

Comment:
can we note somehow that these changes are because of the new symfony project structure and not because we decided to change things arbitrarily?

Reply:
Of course, have added it.